debug/Debug.js

```diff
@@ -0,0 +1,130 @@
+// importing the built version instead of root because dat.gui is badly packaged and exposes ES6
+import dat from 'dat.gui/build/dat.gui';
+import Chart from 'chart.js';
+
+/**
+ * Create a debug instance attached to an itowns instance
+ *
+ * @Constructor
+ * @param {Scene} scene the itowns Scene
+ * @return {Debug} a debug instance
+ */
+// disabling eslint errors as it is the exported constructor
+function Debug(scene) {
+    // CHARTS
+    // create charts div
+    const chartDiv = document.createElement('div');
+    chartDiv.id = 'chart-div';
+    chartDiv.style = 'position: absolute; bottom: 0; left: 0; width: 100vw; height: 20rem; background-color: white; display: none';
+
+    scene.viewerDiv.appendChild(chartDiv);
+
+    // line graph for nb elements
```

Comment:
When you display charts, there's bug with navigation
Mouse navigation does not work properly